### ISSUE TITLE
Inconsistent tab closing behavior makes it easy to accidentally close the whole window

### ISSUE THREAD

**Issue by AbeJellinek** [zotero/zotero#2572]
A couple related bits here:

- Ctrl/Cmd-W on the library tab closes the whole window even if other tabs are open.
- Closing a tab pops back to the library if you haven't switched tabs since opening it, but to another PDF tab if you have. (Browsers pop back to a just-opened tab's opener too but usually open new tabs directly to the right of the opener, which makes it easier to predict which tab will be switched to.)

So it ends up being pretty easy to accidentally close the entire window while repeatedly Cmd-W-ing to close leftover tabs.

[We've gotten reports of accidental window closes](https://forums.zotero.org/discussion/96624/minimize-or-close-to-system-tray), and someone actually wrote a plugin linked in that thread to minimize instead of closing, but I'm not sure if it's specifically because of the behavior outlined here.

**Comment by dstillman** [comment]
> Ctrl/Cmd-W on the library tab closes the whole window even if other tabs are open.

Thunderbird seems to ignore Cmd-W on the first tab if multiple tabs are open. That's probably reasonable for us, at least as long as every window has one and only one library tab in the first position.

> Closing a tab pops back to the library if you haven't switched tabs since opening it, but to another PDF tab if you have.

This is [by design](https://github.com/zotero/zotero/issues/1985), and I think the benefits outweigh any drawbacks, particularly if we block Cmd-W on the first tab.

**Comment by AbeJellinek** [comment]
Right, it's more that the second is why the first ends up being a problem.

**Comment by dstillman** [comment]
https://forums.zotero.org/discussion/100694/order-of-closing-tabs

**Comment by sraedler** [comment]
Is there a rough timeline for when this problem will be fixed?
Closing the application on Ctrl+W is a bit annoying.
